pages/schedule.tsx

```diff
@@ -167,6 +168,8 @@ export default function Schedule({ filters, shifts }: ISchedulesProps) {
           />
         </div>
 
+        <Legend></Legend>
```

Comment:
The type of class doesn't need a legend , because its already displayed inside the Card component.

Reply:
I think it clarifies the use of different colors, otherwise the user may be confused at first.

Reply:
Well as i said before the type of class is being displayed inside the card, so once the user press select a class he can easily read what's inside of it. Otherwise, the Legend is placed too far down and since the filters already have their own scroll, the user won't need to scroll down the page so he won't even notice that the Legend exists.

Reply:
Fair enough, I removed it ;)